python/src/aac/spec/__init__.py

```diff
@@ -1,11 +1,11 @@
 """Provide access to the core AaC spec."""
 
-from aac.spec.core import get_aac_spec, get_aac_spec_as_yaml, get_primitives, get_root_keys, get_root_fields
+from aac.spec.core import get_aac_spec, get_aac_spec_as_yaml, get_primitives, get_root_keys, get_root_definitions
 
 __all__ = (
     get_aac_spec.__name__,
     get_aac_spec_as_yaml.__name__,
     get_primitives.__name__,
     get_root_keys.__name__,
-    get_root_fields.__name__,
+    get_root_definitions.__name__,
 )
```

Comment:
I've got no strong style opinion.  But I'd suggest that this change was done to maintain consistency with the rest of the code base.  Recommend we make any such changes in a single PR rather than sprinkled around.